### PR TITLE
carousel

### PR DESCRIPTION
**Pull request by MostafaRastegar**

### CODE REVIEW

File: packages/ui/packages/Carousel/package.json [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  "module": "dist/index.es.js",
+  "scripts": {
+    "test": "cross-env CI=1 react-scripts test --env=jsdom",
+    "test:watch": "react-scripts test --env=jsdom",
+    "build": "rollup -c",
+    "start": "rollup -c -w",
+    "prepare": "yarn run build"
+  },
+
+  "peerDependencies": {
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/package.json]: package json has error

Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/package.json]: done.

File: packages/ui/packages/Carousel/index.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import React, { useEffect, useRef } from 'react';
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/index.js]: in module it is better to use this syntax :
```js
import * as React from 'react'; 
import { useEffect, useRef } from 'react';
```

File: packages/ui/packages/Carousel/index.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import PropTypes from 'prop-types';
+// import { Slider } from 'es6-simple-carousel';
+import { Slider } from './src/index';
+import { StyledSimpleCarousel } from './src/styles';
+
+function SimpleCarousel(props) {
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/index.js]: it is better to use `const fnName = (props) => `

File: packages/ui/packages/Carousel/src/styles.js [hunk 4]
```diff
@@ -0,0 +1,111 @@
+import Styled from 'styled-components';
+
+export const StyledSimpleCarousel = Styled.div`
+
+.slides {
+    display: flex;
+    position: relative;
+    width: 5000000px;
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/src/styles.js]: are you sure this `width` is required ?

File: packages/ui/packages/Carousel/src/styles.js [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+      z-index: 2;
+      align-items: center;
+      justify-content: center;
+      display:flex;
+      direction: ltr;
+  }
+  .control:after{
+      display: block;
+      font-size: 36px;
+  }
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/src/styles.js]: indent ?

File: packages/ui/packages/Carousel/src/styles.js [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+  
+  .dots li.active {
+    background-color: #efefef;
+  }
+  
+  .slider-rtl{
+      direction: rtl;
+  }
+  .slider-rtl .control{
+    transform: rotateY(180deg);
```
Comment (sayjeyhi) [inline packages/ui/packages/Carousel/src/styles.js]: indent

File: packages/ui/packages/Carousel/index.js (hunk 2 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/index.js]: done

File: packages/ui/packages/Carousel/index.js (hunk 3 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/index.js]: done

File: packages/ui/packages/Carousel/src/styles.js (hunk 4 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/src/styles.js]: it's ok

File: packages/ui/packages/Carousel/src/styles.js (hunk 5 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/src/styles.js]: done

File: packages/ui/packages/Carousel/src/styles.js (hunk 6 above)
Comment (MostafaRastegar) [reply packages/ui/packages/Carousel/src/styles.js]: done

File: packages/ui/packages/Carousel/src/styles.js (hunk 4 above)
Comment (sayjeyhi) [reply packages/ui/packages/Carousel/src/styles.js]: @MostafaRastegar what is that purpose?

**Pull request merged**